src/main/java/frc/robot/commands/ClimbEnable.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
This file isn't used since we use an instantcommand. Delete this file

Reply:
done